modules/cloud-config-container/simple-nva/files/frr/frr.conf

```diff
@@ -0,0 +1,23 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Default FRR configuration which enables a BGP protocol process with a default ASN of 65500.
+# For more information on how to update this file and/or configure bgp sessions please refer to 
+# the official documentation available at the following link:
+# https://docs.frrouting.org/en/latest/overview.html
+
```

Comment:
What's the use case for this? What the router would do just with this file?
I'm thinking that with not even bgp neighbors set this will just sit there doing nothing. Is it worth?
Are you thinking to a user testing bgp in gcp and accessing the frr instance using the cli? If we still want to allow this, should we put an example somewhere in the readme?

Reply:
Use case is bootstrapping the NVA with FRR container and having BGP daemon running on the VM. Pretty similar use case of installing frr from a linux VM, but providing a very basic frr.conf file with basic setup to use as a reference (i could also avoid to pass the file during bootstrap since it is not mandatory for bgp service to start. I see two options:
-  removing the file and keeping super simple (maybe adding an example file in README)
- keep current configuration and provide a blueprint using the NVA with BGP and adding a reference in the REAME section of this file

What do you think?

Reply:
I think this without any config wouldn't make much sense.

I would:
* make the `frr_config` variable mandatory
* remove the `enable` variable, in favor of enabling everything automatically if the outer variable is specified
* rename the bgp_config to something else, given at this point this doesn't just work for bgp but also for other protocols

Reply:
Implemented proposed solution, please review latest version of the module.